backend/controllers/authUserMaintenance.js

```diff
@@ -1,13 +1,16 @@
 const log = require('../log')
 
-const handleClientError = async (apiCall, defaultField, res) => {
+const handleClientError = async (apiCall, defaultField, res, errorMapping) => {
   try {
     await apiCall()
   } catch (e) {
-    if (e.response && e.response.status < 500) {
+    if (e.response && e.response.data && e.response.status < 500) {
       res.status(e.response.status)
-      const field = e.response.data.field ? e.response.data.field : defaultField
-      res.json([{ targetName: field, text: e.response.data.error_description }])
+      // eslint-disable-next-line camelcase
+      const { field, error, error_description } = e.response.data
```

Comment:
meh, could you remove the eslint hint if you use aliasing? it is supposed to ignore it: https://github.com/eslint/eslint/issues/3185

Reply:
Good call.  The problem was that I wasn't using aliasing, just destructuring into a variable of the same name.  Will fix.